server/jobs/scores_audit.py

```diff
@@ -0,0 +1,42 @@
+from collections import defaultdict
+
+from server import jobs
+from server.models import Assignment, Score, db
+
+@jobs.background_job
+def audit_missing_scores(assign_id, omit_details=('autograder', 'regrade')):
```

Comment:
Overall I'm against making scores actually mean something. OK should just treat scores as arbitrary tags, which users can use however they want